controllers/idle_handlers.go

```diff
@@ -126,26 +126,32 @@ func (r *ImageBasedUpgradeReconciler) cleanup(
 		r.Log.Error(err, msg)
 		errorMessage += msg + " "
 	}
-	// Terminate precaching worker thread
+
+	r.Log.Info("Terminating precaching worker thread, will wait up to 30 seconds")
```

Comment:
I love the extra logs. I think we need to be more verbose in general, as the logs will be all we have if we need to debug external issues

Reply:
Yeah putting these logs helped me find an issue without the needing to bring out the debugger/breakpoints enabled build. Wish we had a `.Debug` be have less visual clutter but then again `info` and `debug` sometimes are interchangeable

Reply:
Plus, it's harder to go back and get an additional level of logs later, especially if it's a one-off or rare issue / race condition